File: loop-structures/Balance.java
import java.util.Scanner;
public class Balance{
    public static void main(String[] args){

        Scanner input = new Scanner(System.in);

        while (true) {
            System.out.print("Enter account number or -1 to exit: ");
            int account = input.nextInt();

            if (account == -1) {
                System.out.println("Exiting ");
                break;
                }
    
            System.out.print("Enter beginning balance: ");
            int beginningBalance = input.nextInt();
    
            System.out.print("Enter items charged: ");
            int itemsCharged = input.nextInt();

            System.out.print("Enter credits applied: ");
            int creditApplied = input.nextInt();

            System.out.print("Enter credit limit: ");
            int creditLimit = input.nextInt();

            int newBalance = beginningBalance + itemsCharged - creditApplied;
            System.out.println("Your new balance is " + newBalance);

            if (newBalance > creditLimit){
                System.out.println("Credit Limit exceeded");
                }
            else {
                System.out.println("Credit Limit is okay");
                }





        }  


    }
}

File: loop-structures/balance.py
while True:
    account_number = int(input("Enter account number or -1 to exit: "))
    if account_number == -1:
        break
    else:
        beginning_balance = int(input("Enter beginning balance: "))
        items_charged = int(input("Enter items_charged: "))
        credit_applied = int(input("Enter credit applied: "))
        credit_limit = int(input("Enter credit limit: "))
        
        new_balance = beginning_balance + items_charged - credit_applied
        print("Your new balance is", new_balance)
    
        if new_balance > credit_limit:
            print("Credit limit exceeded")
        else:
            print("Credit limit is okay")
